docs/Installation.md

```diff
@@ -668,10 +671,29 @@ sed -i '' '/[[:<:]]engine[[:>:]]/ s/=.*/= PostgreSQL/' /usr/local/etc/zonemaster
 ```
 > **Note:** See the [backend configuration] documentation for details.
 
-Install, configure and start database engine (and Perl bindings):
+Install, configure and start database engine and Perl bindings:
+
+```sh
+pkg install p5-DBD-Pg
+```
+
+The Perl bindings library (`p5-DBD-Pg`) has a dependency to a specific version
+of `postgresql-client`. Determine what version was installed:
+
+```sh
+pkg info | grep postgresql |grep client

```

Comment:
Maybe add a space after the second pipe?

Reply:
Yes, no technical difference but for consistency. Done.